pkg/csi/service/wcp/controller_test.go

```diff
@@ -259,7 +259,16 @@ func TestWCPCreateVolumeWithStoragePolicy(t *testing.T) {
 // but not storage topology type. It is a negative case.
 func TestWCPCreateVolumeWithZonalLabelPresentButNoStorageTopoType(t *testing.T) {
 	ct := getControllerTest(t)
-
+	err := commonco.ContainerOrchestratorUtility.DisableFSS(ctx, "Workload_Domain_Isolation_Supported")
+	if err != nil {
+		t.Fatal("failed to disable Workload_Domain_Isolation_Supported FSS")
+	}
+	defer func() {
+		err := commonco.ContainerOrchestratorUtility.EnableFSS(ctx, "Workload_Domain_Isolation_Supported")
```

Comment:
Isn't the FSS called `"workload-domain-isolation"`? This name is for capability right?
Can we use `common.WorkloadDomainIsolationFSS`?

Reply:
capability is added as an FSS in https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/1dfe7e2587dead58161000bc94114c2f2f94ae61/pkg/common/unittestcommon/utils.go#L76-L78